frontend/src/Project/service/labels/cellTypesMachine.js

```diff
@@ -323,7 +323,7 @@ const createCellTypesMachine = ({ eventBuses, undoRef }) =>
           const pred = evt.predictions;
           for (let cell in pred) {
             cellTypes = cellTypes.map((cellType) =>
-              cellType.id === pred[cell] + 1 && !cellType.cells.includes(cell)
+              cellType.id === pred[cell] + 1 && !cellType.cells.includes(cell) && cell > 0
```

Comment:
What is the new `cell > 0` check catching? Not a blocker, I'm just curious!

Reply:
Cell 0 is the background, so the background ends up getting labeled by a cell type without it